x-pack/legacy/plugins/maps/public/layers/styles/vector/properties/dynamic_color_property.test.js

```diff
@@ -28,35 +28,56 @@ const mockField = {
   getRootName() {
     return 'foobar';
   },
+  getOrigin() {
+    return FIELD_ORIGIN.SOURCE;
+  },
   supportsFieldMeta() {
     return true;
   },
 };
 
-const getOrdinalFieldMeta = () => {
-  return { min: 0, max: 100 };
-};
+class MockStyleMeta {
```

Comment:
Mocks lead to gaps in coverage as the underlying implementation changes. Mocks should be avoided unless its really painful to instantiate the real thing. Since StyleMeta only takes a descriptor, just use the real class here.